packages/ui/components/AppLayout/Navbar.tsx

```diff
@@ -5,80 +5,125 @@ import { Button } from 'components/Button';
 import { DropDown } from 'components/DropDown';
 import OWNER_DATA from 'fixtures/dropdown.json';
 
-const Navbar = ({ username }: { username: string }) => {
+const Navbar = ({
+  username,
+  setEditProgramModalOpen,
+  setAccountSettingModalOpen,
+}: {
+  username: string;
+  setEditProgramModalOpen: Function;
+  setAccountSettingModalOpen: Function;
+}) => {
   const [open, setOpenDropDown] = useState(false);
   const [isAccountDropDownOpen, setAccountDropDown] = useState(false);
   const [isUserDropDownOpen, setUserDropDown] = useState(false);
+  const [hover, setHover] = useState(false);
+
+  const tooltipStyle = {
+    display: hover ? 'block' : 'none',
+  };
 
   return (
-    <nav className="bg-white py-4 px-8 h-16 flex items-center justify-between">
-      <ul className="flex items-center">
+    <nav className="bg-white py-4 px-8 h-16 flex items-center justify-between border border-gray-100 hover:border-blue-400">
```

Comment:
@pipesort123TP would have to extract out this as a separate class under a module.css